layers/lispy/packages.el

```diff
@@ -0,0 +1,74 @@
+;;; packages.el --- lispy layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author:  <burketje@BURKETJE-2014>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;; See the Spacemacs documentation and FAQs for instructions on how to implement
+;; a new layer:
+;;
+;;   SPC h SPC layers RET
+;;
+;;
+;; Briefly, each package to be installed or configured by this layer should be
+;; added to `lispy-packages'. Then, for each package PACKAGE:
+;;
+;; - If PACKAGE is not referenced by any other Spacemacs layer, define a
+;;   function `lispy/init-PACKAGE' to load and initialize the package.
+
+;; - Otherwise, PACKAGE is already referenced by another Spacemacs layer, so
+;;   define the functions `lispy/pre-init-PACKAGE' and/or
+;;   `lispy/post-init-PACKAGE' to customize the package as it is loaded.
+
+;;; Code:
+
+(defconst lispy-packages
+  '(lispy))
+
+(defun lispy/init-lispy ()
+  (use-package lispy
+    :config
+    (defun spacemacs/lispy-off () (lispy-mode -1))
```

Comment:
If someone turns off lispy mode, exiting the (state)hook will cause this to fire off and based on [this comment](https://github.com/abo-abo/lispy/issues/141#issuecomment-156796735), it's better to guard it to avoid unintended issues.